front/app/components/IdeaCards/shared/Filters/TopicFilterDropdown.tsx

```diff
@@ -1,55 +1,49 @@
-import React, { memo, useState } from 'react';
+import React, { memo, useMemo } from 'react';
```

Comment:
Having memo and useMemo in the same component is a bit strange. Can we remove `memo`?